### PR TITLE
fix llmbench report bug

### PR DESCRIPTION
**Pull request by WeizhuoZhang-intel**

### CODE REVIEW

**Review by ESI-SYD — APPROVED** [review]

**Comment by chuanqi129** [comment]
As I mentioned in the email, eager mode doesn't need cpp_wrapper scenario test. https://github.com/chuanqi129/inductor-tools/blob/yudong/aws_auto/scripts/llmbench/env_prepare.sh#L97C81-L97C93, so the result2 table also need update

File: scripts/llmbench/generate_report.py [hunk 1]
```diff
@@ -104,9 +96,7 @@ def calculate_ratio(values,last_values):
     ratio_inductor_llama_cpp = calculate_ratio(last_latency_dict["inductor_llama_cpp"],latency_dict["inductor_llama_cpp"])
 
     ratio_eager_gtpj_default = calculate_ratio(last_latency_dict["eager_gptj_default"],latency_dict["eager_gptj_default"])
-    ratio_eager_gtpj_cpp = calculate_ratio(last_latency_dict["eager_gptj_cpp"],latency_dict["eager_gptj_cpp"])
```
Comment (chuanqi129) [inline scripts/llmbench/generate_report.py:107]: We may need keep this 2 lines in result 3 speedup table, just replace the baseline `eager_gptj_cpp` as `eager_gptj_default` is enough. similar for llama

File: scripts/llmbench/env_prepare.sh
Comment (chuanqi129) [inline scripts/llmbench/env_prepare.sh:1]: Seems we need update the `collect_perf()` function in here to change the eager baseline. cc: @ESI-SYD

**Review by ESI-SYD — COMMENTED** [review]
Believe `latency_dict["eager_gptj_cpp"] =latency_dict["eager_gptj_default"] `
  So pls update here https://github.com/chuanqi129/inductor-tools/blob/yudong/aws_auto/scripts/llmbench/generate_report.py#L39-L40
```
latency_dict["eager_gptj_cpp"] = result.loc[12,'item']
latency_dict["eager_llama_cpp"] = result.loc[13,'item']
```

File: scripts/llmbench/generate_report.py [hunk 2]
```diff
@@ -36,13 +36,13 @@ def calculate_ratio(values,last_values):
 
 latency_dict["eager_gptj_default"] = result.loc[12,'item']
 latency_dict["eager_llama_default"] = result.loc[13,'item']
-latency_dict["eager_gptj_cpp"] = result.loc[14,'item']
-latency_dict["eager_llama_cpp"] = result.loc[15,'item']
+latency_dict["eager_gptj_cpp"] = result.loc[12,'item']
+latency_dict["eager_llama_cpp"] = result.loc[13,'item']
 
 # inductor speedup: 4 values
 gptj_default_inductor_speedup = result.loc[16,'item'].split(',')[0]
```
Comment (chuanqi129) [inline scripts/llmbench/generate_report.py:43]: Will the `16` still available?

**Comment by chuanqi129** [comment]
Let's change the nightly job based on your PR branch, and check tomorrow nightly report, if there is no issue, we can merge this PR

**Review by chuanqi129 — APPROVED** [review]

**Comment by ESI-SYD** [comment]
@WeizhuoZhang-intel  We also need an update here, [5] == [7] and [6] == [8], https://github.com/chuanqi129/inductor-tools/pull/17/files#diff-3ef294d6020a654cf320fba6321f92658644c594166b05baac810923baee9c61R82-R85

to:
```
    speedup1=`awk 'BEGIN{printf "%.2f",'${infer_latency[5]}' / '${infer_latency[1]}'}'`
    speedup2=`awk 'BEGIN{printf "%.2f",'${infer_latency[6]}' / '${infer_latency[2]}'}'`
    speedup3=`awk 'BEGIN{printf "%.2f",'${infer_latency[5]}' / '${infer_latency[3]}'}'`
    speedup4=`awk 'BEGIN{printf "%.2f",'${infer_latency[6]}' / '${infer_latency[4]}'}'

```

**Comment by ESI-SYD** [comment]
Seems no issue with this change in inductor-nightly report, merged

**Comment by chuanqi129** [comment]
> Seems no issue with this change in inductor-nightly report, merged

I think yesterday nightly not use this PR branch. @ESI-SYD

**Comment by ESI-SYD** [comment]
> > Seems no issue with this change in inductor-nightly report, merged
> 
> I think yesterday nightly not use this PR branch. @ESI-SYD

I see weizhuo have set, @WeizhuoZhang-intel right?
image

**Comment by chuanqi129** [comment]
But seems from the report, I don't see any different with before...

**Comment by ESI-SYD** [comment]
The row 1 change to exactly the same as row 2, also, row 3 same as row 4
`Result 2: use_dynamo = False`

**Comment by chuanqi129** [comment]
> The row 1 change to exactly the same as row 2, also, row 3 same as row 4 `Result 2: use_dynamo = False`

Got you. That means the table same with before, but test already deleted.

**Pull request merged**